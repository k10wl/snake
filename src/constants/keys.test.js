/* eslint-disable */
import bindKeys from "./keys";

describe("Key binding snake controls", () => {
  const fnMock = jest.fn();
  const event = new KeyboardEvent('keydown', {'key': 'a'});
  document.dispatchEvent(event);
  it("should turn left", function() {
    expect(bindKeys(fnMock)).toMatchSnapshot();
  })
});