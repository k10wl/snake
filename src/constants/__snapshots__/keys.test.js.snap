// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Key binding snake controls should turn left 1`] = `undefined`;
